cove/locale/es/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,1035 @@
+"descargar el archivo"
+
+#: cove/input/views.py:70 cove/input/views.py:78 cove/lib/converters.py:89
+#: cove/lib/converters.py:154 cove/lib/exceptions.py:41
+msgid "Try Again"
+msgstr "Inténtalo de nuevo"
+
+#: cove/input/views.py:71
+msgid ""
+"\n"
+"\n"
+" Common reasons for this error include supplying a local development url that our servers can't access, or misconfigured SSL certificates."
+msgstr ""
+"\n"
+"\n"
+"Las razones más comunes de este error incluyen el suministro de una URL desarrollo local que nuestros servidores no pueden acceder o certificados SSL mal configurados."
+
+#: cove/input/views.py:76
+msgid "Sorry we got a HTTP Error whilst trying to download that file"
+msgstr ""
+"En este momento tenemos un error de HTTP al mismo tiempo tratando de "
+"descargar el archivo"
+
+#: cove/input/views.py:79
+msgid ""
+"\n"
+"\n"
+" If you can access the file through a browser then the problem may be related to permissions, or you may be blocking certain user agents."
+msgstr ""
+"\n"
+"\n"
+"Si se puede acceder al archivo a través de un navegador entonces el problema puede estar relacionado con los permisos, o puede estar bloqueando ciertas aplicaciones de usuario."
+
+#: cove/lib/converters.py:87 cove/lib/converters.py:152
+#: cove/lib/exceptions.py:39
+msgid "Sorry we can't process that data"
+msgstr "Lo sentimos, no podemos procesar esos datos"
+
+#: cove/lib/converters.py:90
+msgid ""
+"We think you tried to supply a spreadsheet, but we failed to convert it to JSON.\n"
+"\n"
+"Error message: {}"
+msgstr ""
+"Creemos que se trató de suministrar una hoja de cálculo, pero no hemos podido convertirlo a JSON.\n"
+"\n"
+"Mensaje de error: {}"
+
+#: cove/lib/converters.py:155
+msgid ""
+"We think you tried to upload a JSON file, but it is not well formed JSON.\n"
+"\n"
+"Error message: {}"
+msgstr ""
+"Pensamos que ha intentado cargar un archivo JSON, pero no es un JSON bien formado.\n"
+"\n"
+"Mensaje de error: {}"
+
+#: cove/lib/exceptions.py:42
+msgid ""
+"We did not recognise the file type.\n"
+"\n"
+"We can only process json, csv and xlsx files."
+msgstr ""
+"No se reconoce el tipo de archivo. \n"
+"\n"
+"Sólo podemos procesar archivos JSON, CSV y xlsx."
+
+#: cove/templates/500.html:8
+msgid "Internal Server Error"
+msgstr "Error de Servidor Interno"
+
+#: cove/templates/500.html:12
+msgid ""
+"Something went wrong with your request. This could be due to a bug in the "
+"application and/or the data you supplied."
+msgstr ""
+"Algo salió mal con tu solicitud. Esto podría ser debido a un error en la "
+"aplicación y / o los datos que proporcionaste."
+
+#: cove/templates/500.html:14
+msgid ""
+"This error has been automatically reported to our development team. If you "
+msgstr ""
+"Este error se ha informado de forma automática a nuestro equipo de "
+"desarrollo. Si tu"
+
+#: cove/templates/500.html:14
+msgid "please reference this error as "
+msgstr "por favor haz referencia a este error como"
```

Comment:
The Spanish translation is a bit inconsistent as it mixes formal ('usted) and informal ('tú') manners of addressing users. I tried to follow suit and use the formal one as that is what I saw in previous translations, but as exemplified in this line it is not consistent.